feat(changelog): messaging-and-queuing-added-queues-subscription-availab 2024-10-08 (#3814)
scaleway/docs-content: changelog/october2024/2024-10-08-messaging-and-queuing-added-queues-subscription-availab.mdx

---

> title: Queues subscription available for Messaging and Queuing Topics

Reviewer: ```suggestion
title: Queue subscriptions available for Messaging and Queuing topics
```

status: added
author:
    fullname: 'Join the #messaging-queuing channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-10-08
category: serverless
product: messaging-and-queuing
---

> You can now subscribe directly to queues from your topics, available on the console and the API for standard and FIFO topics. 

Reviewer 2: ```suggestion
You can now subscribe directly to queues from your topics, from the [Scaleway console](https://console.scaleway.com), and via the [API](https://www.scaleway.com/en/developers/api/) for standard and FIFO topics. 
```
